index.js

```diff
@@ -830,13 +838,14 @@
                       reprs.join(_(', ')));
 
     var padding = _(showTypeSig_(expTypes.slice(0, index)));
```

Comment:
(also might want to add a test for pointing to return values, which is how I discovered something might be amiss..)